### ISSUE TITLE
Using colabcode on Colab is not safe  now?

### ISSUE THREAD

**Issue by Wallfacer005CN** [abhishekkrthakur/colabcode#115]
When I tried to run colabcode in Colab , I got the warning:
 
The code you are executing may not be allowed, which may restrict your use of Colab later. Note the forbidden actions listed in our FAQ.

Does this mean colabcode is not safe now?

**Comment by abhishekkrthakur** [comment]
not really, nothing has changed in our codebase. google colab has been blocking tunneling, thats it :)

**Comment by Wallfacer005CN** [comment]
I'm sorry I didn't make myself clear and you misunderstood. I don't mean that this project has a dangerous code, but I want to confirm whether there is a risk that I will be banned by colab if I use this project in Colab.

**Comment by abhishekkrthakur** [comment]
That is something i cant answer, unfortunately.
